HW_7/view.py

```diff
@@ -0,0 +1,105 @@
+
+from logger import log
+
+
+@log
+def greatings():
+    '''Вывод приветствия.'''
+
+    print('Справочник')
+    pass
+
+
+@log
+def menu() -> int:
+    """
+    Вывод меню, запрос данных от пользователя.
+    :return:
+    0 - Выход
+    1 - Загрузить из файла и вывести на экран
+    2 - Добавить новую запись
+    3 - Редактировать запись по id
+    4 - Поиск по фамилии

```

Comment:
Так как это все попадет в лог, запись не самая удачная, слишком много инфы
```suggestion
    вывод меню
```